controllers/seedimage_controller.go

```diff
@@ -465,32 +459,7 @@ func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pul
 			},
 		},
 		Spec: corev1.PodSpec{
-			InitContainers: []corev1.Container{
-				{
-					Name:            "build",
-					Image:           buildImg,
-					ImagePullPolicy: pullPolicy,
-					Resources: corev1.ResourceRequirements{
-						Limits: corev1.ResourceList{
-							corev1.ResourceEphemeralStorage: *resource.NewQuantity(volLim, resource.BinarySI),
-						},
-						Requests: corev1.ResourceList{
-							corev1.ResourceEphemeralStorage: *resource.NewQuantity(volRes, resource.BinarySI),
-						},
```

Comment:
Well. you actually dropped all the ResourceRequirements: this here was just for scheduling, to ensure the destination node has enough disk space to be picked up to start the node (which actually will be the space reserved for the shared volume).

Reply:
I thought this was apart from the shared volume. I could be easily misunderstanding this feature ResourceRequirements here. I thought this ResourceRequirement here was only for the base image, so I assumed the Pod requirements are the shared volume plus this ResourceRequirement here. That's why I thought about downloading the base image to the shared volume, this way the resource constraints can be expressed in the same way regardless downloading it form HTTP or pulling it as a container, as in both cases the base image is copied/downloaded into the shared volume.

This is the reasoning of the change, but probably I am misunderstanding it.

Of course we adjust the share volume size to whatever we consider, to my understanding to be fully consistent with the former code it should be set to 8GiB (twice the max base image size).